lib/onfido/resource.rb

```diff
@@ -109,25 +96,22 @@ def general_api_error(response_code, response_body)
       )
     end
 
-    def handle_restclient_error(error, url) # rubocop:todo Metrics/MethodLength
+    def handle_restclient_error(error) # rubocop:todo Metrics/MethodLength
```

Comment:
Made some minor changes to error messages so they don't need the URL or global config any more.